### PR TITLE
Create mathable module

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ahenzler — APPROVED** [review]
Great helper methods for our managers. I think this will be really useful.

**Review by Aphilosopher30 — APPROVED** [review]
looks like a good start

File: lib/mathable.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module Mathable
```
Comment (denverdevelopments) [inline lib/mathable.rb:1]: Very useful module methods

File: lib/games_manager.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
+require_relative './mathable'
 require_relative './game'
```
Comment (denverdevelopments) [inline lib/games_manager.rb:2]: Good module implementation

**Review by denverdevelopments — APPROVED** [review]

**Pull request merged**